artview/core/core.py

```diff
@@ -78,10 +78,14 @@ class Variable(QtCore.QObject):
         |VplotAxes  | Hold axes of      |:py:class:`matplotlib.axes.Axes`    |
         |           | Matplotlib figure |instance or None                    |
         +-----------+-------------------+------------------------------------+
-        |VPathInte\ | Hold auxiliar     |function like :py:func:`~artview.\  |
+        |VPathInte\ | Hold auxillary    |function like :py:func:`~artview.\  |
         |riorFunc   | function          |components.RadarDisplay.\           |
         |           |                   |getPathInteriorValues` or None      |
         +-----------+-------------------+------------------------------------+
+        |Vfilelist  | Hold filenames in |list containing string file         |
+        |           | current working   |basenames                           |
```

Comment:
if file-list just have basenames, how does the components know the directory? and can't I mix files from different directories?